recipes/scikit-sparse/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.3" %}
+
+package:
+  name: scikit-sparse
+  version: {{ version }}
+
+source:
+  fn: scikit-sparse--{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-sparse/scikit-sparse-{{ version }}.tar.gz
+  sha256: 2456be16e65bb8126ea7b9a3b175282b396d07ebeb0b4c5ac85d707bfc977008
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - suitesparse
+    - cython
+    - setuptools
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - suitesparse
+
+test:
+  requires:
+    - nose
+    - setuptools
+  imports:
+    - sksparse
+    - sksparse.cholmod
+  commands:
+    - nosetests -sv sksparse
+
+about:
+  home: https://github.com/scikit-sparse/scikit-sparse
+  license: BSD 2-clause
+  summary: A companion to the scipy.sparse library for sparse matrix manipulation in Python.
+
+extra:
+    recipe-maintainers:
+      - grlee77
```

Comment:
Spacing is a little funky here, but it is not a blocker. Feel free to address this later.